### PR TITLE
[Keyboard] Whitefacemountain Ampersand

### PR DESCRIPTION
**Pull request by roarmstrong**
## Description

Adds definitions for the Whitefacemountain Ampersand keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/whitefacemountain/ampersand/README.md
Comment (waffle87) [inline keyboards/whitefacemountain/ampersand/README.md:1]: This file must be renamed to lowercase.

File: keyboards/whitefacemountain/ampersand/info.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    "keyboard_name": "Ampersand",
+    "maintainer": "roarmstrong",
+    "bootloader": "atmel-dfu",
+    "bootmagic": {
+        "enabled": true,
+        "matrix": [0, 0]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true
```
Comment (waffle87) [inline keyboards/whitefacemountain/ampersand/info.json:12]:
```suggestion
        "bootmagic": true,
        "mousekey": true,
        "extrakey": true

File: keyboards/whitefacemountain/ampersand/info.json [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        "rows": ["F5", "F0", "C7", "F6"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://www.cbkbd.com/product/ampersand",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x2626",
+        "vid": "0xFEED"
+    },
+    "layouts": {
```
[the remark this answers is not in the document]
Comment (roarmstrong) [reply keyboards/whitefacemountain/ampersand/info.json]: Thank you! I think I've updated the layouts correctly now.

**Review by waffle87 — APPROVED** [review]

File: keyboards/whitefacemountain/ampersand/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[0] = LAYOUT_all(
+	KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_MINS, KC_BSPC,
+	KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+	KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+	KC_LCTL, KC_LALT, KC_LGUI,                   KC_SPC,  KC_SPC,                    KC_RGUI, KC_RALT, KC_RCTL)
```
Comment (drashna) [inline keyboards/whitefacemountain/ampersand/keymaps/default/keymap.c:11]:
```suggestion
    KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_MINS, KC_BSPC,
    KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
    KC_LCTL, KC_LALT, KC_LGUI,                   KC_SPC,  KC_SPC,                    KC_RGUI, KC_RALT, KC_RCTL)
```

**Review by drashna — APPROVED** [review]
Nit:

**Pull request merged**